workalendar/core.py

```diff
@@ -765,7 +765,7 @@ def get_variable_days(self, year):
                 jd = conversion_method(y, month, day)
                 g_year, g_month, g_day = self.calverter.jd_to_gregorian(jd)
                 if g_year == year:
-                    holiday = date(g_year, g_month, g_day)
+                    holiday = date(g_year, g_month, g_day) - timedelta(days=1)
```

Comment:
I can't see why you need to substract one day for ALL Islamic Holidays. Does that only happen in Turkey's calendar? How is it happening that there's a delta?

Reply:
This is the situation in Turkey. And yes, It is since forever. You can examine Turkey holidays of past years and compare it with normal islamic calendar. https://www.timeanddate.com/holidays/turkey/

Reply:
Thanks about this insight.

So now I think I'm getting it. I think I'm going to take over your branch ; I've got an idea, but it needs to refactor the core module. Hopefully there's a solution.
I'll keep you posted.

Reply:
Thank you for this project it is very useful.